suite-common/wallet-utils/src/__tests__/transactionUtils.test.ts

```diff
@@ -24,7 +24,7 @@ describe('transaction utils', () => {
     });
 
     it('parseTransactionMonthKey', () => {
-        expect(parseTransactionMonthKey('2023-01')).toEqual(new Date(2023, 0));
+        expect(parseTransactionMonthKey('2023-00')).toEqual('January');
```

Comment:
`January` should be equal to `2023-01`.

`function const generateTransactionMonthKey = (d: Date) => ${d.getFullYear()}-${d.getMonth() + 1}`
is adding +1 to the month value. You should subtract -1 inside the `parseTransactionMonthKey` function as it was. Then the test will pass without modification.

Now, all months are incorrectly shifted by one.

Reply:
Good catch.

Reply:
Yes. Problem is you didn't add month in localisation function which resulted in different month in the UI as discussed during pair programming. 

So again for @Nodonisko, I created formatter which returns month name and in that formatter I'm doing month addition so UI reflects real data.